ouster_client/src/client.cpp

```diff
@@ -380,6 +464,55 @@ std::shared_ptr<client> init_client(const std::string& hostname,
     return cli;
 }
 
+std::shared_ptr<client> init_client(const std::string& hostname,                                    
+                                    const std::string& udp_dest_host,
+                                    const std::string& mtp_dest_host,
+                                    lidar_mode ld_mode, timestamp_mode ts_mode,
+                                    int lidar_port, int imu_port,
+                                    int timeout_sec) {
+
+    logger().info("initializing sensor: {} with ports: {}/{}, multicast group: {}",
+                  hostname, lidar_port, imu_port, udp_dest_host);
+
+    auto cli = std::make_shared<client>();
+    cli->hostname = hostname;
+
+    cli->lidar_fd = mtp_data_socket(lidar_port, udp_dest_host, mtp_dest_host);
+    cli->imu_fd = mtp_data_socket(imu_port); // no need to join multicast group
```

Comment:
May I ask why not be that case?

Reply:
Because of this I start the 2nd client I don't get IMU messages, I also tried to change the line to `cli->imu_fd = mtp_data_socket(imu_port, udp_dest_host, mtp_dest_host);` yet I still didn't get imu packets on the 2nd client, any idea why?

Reply:
I can check it and try to fix it tomorrow

Reply:
Can you check it again? Cause i can see imu packets on both clients now, main and slave. May be you configuration is wrong? 
I check it at this [commit of sdk](https://github.com/ouster-lidar/ouster_example/pull/480/commits/cf2892afbd626ec057880a119d27029753511203) and [commit of driver](https://github.com/ouster-lidar/ouster-ros/pull/57/commits/c6d8d4e4eebfb4ad60e19973dafd6b2423b535d5).

My launch files for master and slave are attached (renamed to txt)
[sensor_mtp_main.launch.txt](https://github.com/ouster-lidar/ouster_example/files/10732570/sensor_mtp_main.launch.txt)
[sensor_mtp_slave.launch.txt](https://github.com/ouster-lidar/ouster_example/files/10732571/sensor_mtp_slave.launch.txt)

By the way - we dont need to join to multicast group, because we already in it after joining in function for lidar packets. No need to do it twice.